x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.scss

```diff
@@ -59,6 +59,12 @@
   }
 }
 
+.lnsLayerPanel__rowOptional {
+  font-weight: $euiFontWeightLight;
+  font-size: $euiFontSizeXS;
+  color: $euiTextSubduedColor;
+}
```

Comment:
If taking my advice to switch to an `EuiText` in the previous comment, these styles can be removed.